using-mobile-extensions/adobe-campaign-standard/adobe-campaignclassic-api-reference.md

```diff
@@ -0,0 +1,70 @@
+# Campaign API reference
+
+## Set linkage fields
+
+Allows the Campaign to connect fields from separate databases to create a more developed and personalized messaging experience for their end user. Profile template based messages that can contain PII based personalization will be downloaded. 
```

Comment:
- Please change the first sentence so that it reads: "Allows Campaign to connect fields from separate databases and create a more developed and personalized messaging experience. "

- Please change the 2nd sentence so that it reads: "Profile template-based messages that contain PII-based personalization are downloaded. "